modules/admin_manual/pages/configuration/server/occ_commands/app_commands/richdocuments.adoc

```diff
@@ -0,0 +1,82 @@
+= Collabora Online / Secure View
+
+Marketplace URL: {oc-marketplace-url}/apps/richdocuments[Collabora Online]
+
+Collabora Online allows you to work with all kinds of collabora office documents inside ownCloud for improved productivity. For details please see the
+xref:enterprise/collaboration/collabora_secure_view.adoc[Collabora Online / Secure View] documentation.
+
+`config:app` commands to configure the Collabora Online app.
+
+[source,console]
+----
+ config:app:get|set
+ richdocuments wopi_url            WOPI Server URL
+ richdocuments secure_view_option  Enable Secure View
+ richdocuments watermark_text      Watermark pattern displayed in the document
+ richdocuments secure_view_open_action_default  Open documents in Secure View with watermark by default
+----
+
+== App Configuration
+
+All app configurations are set and queried with the `config:app` command set. The examples below set a value. To query a value use `config:app:get` and the corresponding key without any options or attributes.
+
+== Set the WOPI URL
+
+WOPI Server URL
+
+This command sets the WOPI Server to `[IP/URL]:port` +
+`[IP/URL]:port` can be any IP/URL plus the port how the WOPI server can be accessed like `'a.b.c.d'` or `'http:\\a.b.c.d:8098'` or `'https:\\rd.yourdomain.com'` ect. Adding the port is only necessary when not using standard ports.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set richdocuments wopi_url --value='[IP/URL]:port'
+----
+
+== Set the secure_view_option
+
+Enable Secure View (possible values: true/false, default: false)
+
+The following example command enables or disables secure view globally in the system
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set richdocuments secure_view_option --value='true'
+----
+
+== Set the watermark_text
+
+Watermark pattern displayed in the document. It can be arbitrarily string. Including keyword \{viewer-email} would be replaced with current user in the document session
```

Comment:
```suggestion
A watermark pattern is displayed in the document when it is viewed. It can be an arbitrary string. Include the keyword \{viewer-email} will be replaced with the current user in the document watermark
```

Does it put the current user's username or their email address in the watermark?

Reply:
@mrow4a can you give us input?

Reply:
"Included keyword \{viewer-email} will be replaced with the current user in the document watermark"

Reply:
OK - maybe that is correct and we have to document it like that. But it seems strange that the substitution variable is called `viewer-email` but it actually substitutes the user name.

If my username is "phild", my display name is "Phil Davis" and my email address is "phil@example.com" then the watermark has "phild" in it. Correct?

Reply:
Just for this text line a suggestion
`...will be replaced with the current user...` -->
`...will be replaced with the current viewing user...`
And dependent on the answer of @mrow4a , I would add a explanation describing what @phil-davis has highlighted.

Reply:
@phil-davis if email is not available, displayname is shown -> https://github.com/owncloud/richdocuments/blob/master/lib/Controller/DocumentController.php#L919-L923

Reply:
😄 clarifies things, needs more text 😄

Reply:
I updated my suggestion in the first post above. Please review.